File: lua/refactoring/tests/refactor/106/cpp/class-function/extract.start.cpp
#include <iostream>

class Coconut {

  void simpleFunction(int a) {
    int test = 1, test_other = 11;
    for (int x = 0; x < test_other + test; x++) {
      std::cout << x << " " << a << std::endl;
    }
  }
};

File: lua/refactoring/tests/refactor/106/csharp/class-function/extract.start.cs
namespace Test
{
    class Extract
    {
    	public static void simpleFunction(int a)
      	{
        	int test = 1;
			int test_other = 11;
        	for (int x = 0; x < test_other + test; x++) {
          		Console.WriteLine(x + " " + a);
	        }
      	}
    }
}
